GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -61,12 +61,9 @@ protected override void PreCreateEnlistment()
 
             if (!this.SkipMountedCheck)
             {
-                using (NamedPipeClient pipeClient = new NamedPipeClient(GVFSPlatform.Instance.GetNamedPipeName(enlistmentRoot)))
+                if (this.IsExistingPipeListening(enlistmentRoot))
                 {
-                    if (pipeClient.Connect(500))
-                    {
-                        this.ReportErrorAndExit(tracer: null, exitCode: ReturnCode.Success, error: "This repo is already mounted.");
-                    }
+                    this.ReportErrorAndExit(tracer: null, exitCode: ReturnCode.Success, error: $"Process appears to be mounted for {enlistmentRoot}.");

```

Comment:
Similar comment here, I'm not sure that we want to say that the "Process" appears to be mounted.  Maybe something like:

- $"The repo `{enlistmentRoot}` is already mounted"? 

or 

- $"A GVFS.Mount process is already running for {enlistmentRoot}"

(Or I think you could just leave this message unchanged too)